homeassistant/components/harmony/strings.json

```
@@ -41,5 +41,15 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_switches": {
+      "title": "The Logitech Harmony switch platform is being removed",
+      "description": "Using the switch platform to change the current activity is now deprecated and will be removed in a future version of Home Assistant.\n\nHarmony has always been a perfect candidate for the select entity in a time when the select entity did not exist. Turning on one activity turns off all others as that is how Harmony was designed. Multiple activities at once was never possible.\n\nPlease adjust any automations or scripts that you may have to use the select entity that is already available to fix this issue."
```

This can be shortened a bit:
```suggestion
      "description": "Using the switch platform to change the current activity is now deprecated and will be removed in a future version of Home Assistant.\n\nPlease adjust any automations or scripts that use switch entities to instead use the select entity."
```

---

So for now, the suggestion will be to use the remote entity. The select entity may be visited at another time but I do see the convenience for people to use the select entity in the dashboard.